yeoman-generator/index.d.ts

```diff
@@ -68,6 +68,7 @@ declare class Base extends EventEmitter {
     appname: string;
     config: Base.Storage;
     fs: Base.MemFsEditor;
+    options: any;
```

Comment:
Wouldn't that not compile in use in a generator though? It wouldn't be able to access a property like `this.options.debugFlag`